src/executor/LanceurExecutor.java

```diff
@@ -28,11 +32,9 @@ public Integer call() throws Exception {
 		ExecutorService execs = Executors.newSingleThreadExecutor();
 
 		Future<Integer> f = execs.submit(monCallable);
-		System.out.println("suite");
 		try {
 			Integer retour = f.get();
-			System.out.printf("j'ai eu le retour %s%n", retour);
-
+			log.log(Level.INFO, "j'ai eu le retour %s%n", retour);
 		} catch (InterruptedException e) {
 			e.printStackTrace();
 		} catch (ExecutionException e) {
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Combine this catch with the one at line 38, which has the same body. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS2147)